### ISSUE TITLE
gittyup frequently asks for https access data

### ISSUE THREAD

**Issue by borisneubert** [Murmele/Gittyup#538]
GittyUp 1.3.0 on current Windows 10 Pro

My repository is local with a remote http (not https) origin. I am frequently asked for https access data. The dialog box does not mention the URL the http access data are for. I need to click the cancel button several times to make it disappear. Though, the dialog box reappears after some minutes.

Dialog box should show the URL.
Dialog box should not reappear after the cancel button was clicked.

![grafik](https://user-images.githubusercontent.com/8795819/235137188-baa1c162-dc25-4d46-819f-45c0d09bb9a4.png)

**Comment by borisneubert** [comment]
related to #524 and #529?

**Comment by exactly-one-kas** [comment]
Yes, this is a duplicate of #524, but I'm not sure how this relates to #529?
